README.md

```diff
@@ -1,2 +1,35 @@
 # cpma [![Build Status](https://travis-ci.com/fusor/cpma.svg?branch=master)](https://travis-ci.com/fusor/cpma) [![Maintainability](https://api.codeclimate.com/v1/badges/aac7d46fd7899042ce52/maintainability)](https://codeclimate.com/github/fusor/cpma/maintainability)
 Control Plane Migration Assistance:  Intended to help migration cluster configuration of a OCP 3.x cluster to OCP 4.x
+
+
+# Build
+
+```console
+$ make
+$ ./bin/cpma
+```
+
+# Usage
+
+Flags:
+```
+--config string           config file (default is $HOME/.cpma.yaml)
+--debug                   show debug ouput
+-h, --help                help for cpma
+-o, --output-dir string   set the directory to store extracted configuration.
+```
+
+You can find example config in `examples/` 
+
+Example:
+
+```console
```

Comment:
command line or cli.

Reply:
This is an annotation, it won't be rendered